backoffice/modules/catalog/models/ProductVariationPost.ts

```diff
@@ -1,4 +1,5 @@
 export type ProductVariationPost = {
+  id?: number;
```

Comment:
Should create type ProductVariationPut and extends ProductVariationPost like this:

export type ProductVariationPut = ProductVariationPost & { id?: number; }

Reply:
I fixed, please reivew again